modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectEntryLocalServiceTest.java

```diff
@@ -3375,6 +3385,24 @@ public void testUpdateObjectEntry() throws Exception {
 	public void testUpdateObjectEntryWithJavaDelegateObjectValidationRule()
 		throws Exception {
 
+		BiFunction<Map<String, Object>, String, String> biFunction =
+			(inputObjects, sourceName) -> {
+				if (inputObjects.containsKey(sourceName) &&
```

Comment:
Is this condition working properly? Checking the old logic, it doesn't seem to be correct.